modules.d/01systemd-bsod/module-setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# This file is part of dracut.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Prerequisite check(s) for module.
+check() {
+    # If the binary(s) requirements are not fulfilled the module can't be installed
+    require_binaries "$systemdutildir"/systemd-bsod || return 1
+
+    # Return 255 to only include the module, if another module requires it.
+    return 255
+}
+
+# Module dependency requirements.
+depends() {
+    # This module has external dependency on other module(s).
+    echo systemd
```

Comment:
`systemd-bsod.service` is just continuously watching the journal, even its source code is [on the same level of hierarchy](https://github.com/systemd/systemd/tree/main/src/journal), so it should depend only on `systemd-journald`.

We should start actually using the available 01systemd modules in dracut and break out the 00systemd monster. Maybe I submit a PR with that.